content/en/docs/about/contributing.md

```diff
@@ -51,6 +51,8 @@ Follow these instructions if you want to
 
 ### Joining the Kubeflow GitHub Org
 
+- **Note**: Anyone can contribute to Kubeflow, adding yourself as a member in [org.yaml](https://github.com/kubeflow/internal-acls/blob/master/github-orgs/kubeflow/org.yaml#L19) is not a mandatory step.
```

Comment:
Let's use the file link, rather than the link tied to specific lines 

`https://github.com/kubeflow/internal-acls/blob/master/github-orgs/kubeflow/org.yaml`

Reply:
ok I will do that..